start_up.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import sys
+import time
+
+import settings
+import classes
+
+def main():
+    with open(sys.argv[1]) as file:
+        solar_car_processes = [classes.SolarCarProcess(path) for path in file.read().splitlines()]
+
+    # Check processes and respond accordingly
+    while True:
+        for solar_car_process in solar_car_processes:
+            if solar_car_process.check_status():
+                solar_car_process.start()
+                solar_car_process.timesRestarted += 1
```

Comment:
SolarCarProcess can probably keep track of it's own timesRestarted, since it knows everytime it's `start()` function is called.